CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+## Contributing to Swagger-UI
+
+We love contributions from our community of users! This document explains our guidelines and workflows. Please take care to follow them, as it helps us keep things moving smoothly.
+
+#### Environment setup
+
+0. Install Node.js (4 or newer) and npm (3 or newer).
+1. Fork Swagger-UI, and clone your fork.
```

Comment:
@ponelat intended to write: Make a fork of Swagger-UI on GitHub, then clone your fork to your machine.

Will change for clarity, thx!